planet/http.py

```diff
@@ -413,6 +415,29 @@ async def stream(
         finally:
             await response.aclose()
 
+    def client(self,
+               name: Literal['data', 'orders', 'subscriptions'],
```

Comment:
this is great and is something I'd love to see used more broadly in the codebase

Reply:
@jreiberkyle I've been slow to catch on to this use of type hints. I looked into whether the list of strings could be dynamically determined there or whether we could import a type alias from `planet.clients`, but it gets complicated.